HippoMocks/CMakeLists.txt

```diff
@@ -1,12 +1,24 @@
 
 add_library(HippoMocks INTERFACE)
-target_include_directories(HippoMocks INTERFACE .)
+target_include_directories(HippoMocks INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}> $<INSTALL_INTERFACE:include>)
 if (NOT WIN32)
   target_compile_options(HippoMocks
     INTERFACE
       -Wall -Wextra -pedantic -Wno-long-long -std=c++11
```

Comment:
This could lead to build failures if project is built once and used on different platforms (which is a possiblility given that it's header-only). Consider moving flags to HippoMocksTest project instead (where they are useful) to avoid forcing them on other people, and setting `CXX_STANDARD` + `CXX_STANDARD_REQUIRED` properties here instead of `-std=c++11`.

Reply:
This needs to be a `cxx_std_11` compile feature since aforementioned properties aren't supported on interface targets :(